Assets/MixedRealityToolkit.Examples/Demos/Audio/Scripts/obj.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
This (and other non-scene system meta files) are unrelated and should not be getting created / modified.